tests/garage/tf/models/test_gaussian_mlp_model.py

```diff
@@ -11,99 +12,119 @@ def setUp(self):
         super().setUp()
         self.input_var = tf.placeholder(tf.float32, shape=(None, 5))
         self.obs = np.ones((1, 5))
+        self.output_dim = 1
```

Comment:
i think you should test over output_dim = 1 and output_dim > 1 (e.g. you can use a parameterized test)

lots of bugs pop up when people fail to consider both cases.

Reply:
this parameterized?
https://pypi.org/project/parameterized/

Reply:
If you don't want to add it to the dep, you can use [parameterized tests in nose2](https://nose2.readthedocs.io/en/latest/params.html).